src/algokit/core/sandbox.py

```diff
@@ -116,7 +116,14 @@ def ps(self, service_name: str | None = None) -> list[dict[str, Any]]:
         )
         if run_results.exit_code != 0:
             return []
-        data = json.loads(run_results.output)
+
+        # `docker compose ps --format json` on version < 2.21.0 outputs a JSON arary
+        if run_results.output.startswith("["):
+            data = json.loads(run_results.output)
+        # `docker compose ps --format json` on version >= 2.21.0 outputs seperate JSON objects, each on a new line
+        else:
+            data = [json.loads(line) for line in run_results.output if line]
```

Comment:
```suggestion
            data = [json.loads(line) for line in run_results.output.splitlines() if line]
```

Reply:
@joe-p  Just double checked on mac, produces `Unhandled JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)` unless splitlines are applied. Otherwise lgtm - will merge as soon as you commit suggestion, and thanks for tackling this 👍